_layouts/sidebar.md

```diff
@@ -8,6 +8,7 @@
 - Journeys
 
     - [Aayushi Mittal](../journeys/Aayushi-Mittal.md)
+    - [Abhay Nandan Singh](../journeys/deltaTH.md)
```

Comment:
```suggestion

```

Please don't make changes to `_layouts/sidebar.md` file. The gh-action will create a link to your journey on the sidebar.